### DISCUSSION TITLE
Quantification of known spike-in heavy peptides

### DISCUSSION THREAD

**Discussion by Munchic** [vdemichev/DiaNN#404] [Q&A]
Hi Vadim,

We have a set of peptides with heavy K, R (not all K and R are heavy) that we want to include in the spectral library generation to use during DIA search and quantification. The exact sequences and locations of the modifications are known. An issue we ran into is that some spike-in peptides might have two K at different locations but only one of them is heavy. What is a correct way to include this information in DIA-NN? Thank you for your help!

**Comment by vdemichev** [comment]
Hi, the way to do it is by 'in silico' labelling an unlabelled library using --lib-fixed-mod: add 'SILAC' label (mass 0.0) to all lysines and arginines. Then, remove from the resulting library all labels which are located where they are not supposed to be - in R or Python. Then proceed with DIA-NN analysis using the --channels command as usual.

**Reply by Munchic** [reply]
Great, thank you so much!
